SDK-v4/gdpr/README.md

```diff
@@ -0,0 +1,27 @@
+# GDPR Compliant example
+
+Sample project demonstrating the integration of Localytics for a customer supporting GDPR on SDK versions below SDK v5.1.
```

Comment:
Sample project demonstrating the integration of Localytics for a customer supporting privacy requirements and regulations such as GDPR on SDK versions below SDK v5.1.